lockc/src/sysutils.rs

```diff
@@ -0,0 +1,58 @@
+use std::{
+    fs::File,
+    io::{self, prelude::*},
+    path::Path,
+};
+
+#[derive(thiserror::Error, Debug)]
+pub enum CheckBpfLsmError {
+    #[error("regex compilation error")]
+    Regex(#[from] regex::Error),
+
+    #[error("I/O error")]
+    IO(#[from] io::Error),
+
+    #[error("BPF LSM is not enabled")]
+    BpfLsmDisabled,
+}
+
+/// Checks whether BPF LSM is enabled in the system.
+pub fn check_bpf_lsm_enabled<P: AsRef<Path>>(sys_lsm_path: P) -> Result<(), CheckBpfLsmError> {
+    let rx = regex::Regex::new(r"bpf")?;
```

Comment:
This is used only once, at bootstrap. Am I correct? If that's not the case, you might want to make this regexp a static using `lazy_static`, just to same some cycles.

Reply:
It's used only once on the daemon startup. So I think it's fine to leave it as it is.

Reply:
Then everything is good :+1: